README.md

```diff
@@ -40,16 +40,12 @@ import (
 )
 
 func main() {
-	config := minio.Config{
-		Endpoint:        "https://s3.amazonaws.com",
-		AccessKeyID:     "YOUR-ACCESS-KEY-HERE",
-		SecretAccessKey: "YOUR-PASSWORD-HERE",
-	}
-
-	// Default is Signature Version 4. To enable Signature Version 2 do the following.
-	// config.Signature = minio.SignatureV2
+	// Requests are always secure by default. set inSecure=true to enable insecure access.
```

Comment:
// Requests are always secure (HTTPS) by default. Set inSecure=true to enable insecure (HTTP) access.